### PR TITLE
[FAB-72] Setup achievements

### PR DESCRIPTION
**Pull request by daystram**

### CODE REVIEW

File: fableous-fe/src/containers/HubCanvasPage.tsx [hunk 1]
```diff
@@ -186,14 +189,17 @@ export default function HubCanvasPage() {
       ControllerRole.Story,
       ControllerRole.Character,
       ControllerRole.Background,
-    ].every((role) => role in joinedControllers);
+    ].some((role) => role in joinedControllers);
```
Comment (nicolauscg) [inline fableous-fe/src/containers/HubCanvasPage.tsx:192]: just want to check whether this is commited intentionally or not, Im fine with either using `.every()` or `.some()`

**Review by nicolauscg — COMMENTED** [review]
looks good, one qs, will we have popups on student side when they got an achivement, if so, will we do that by the hub sending a ws message?

File: fableous-fe/src/containers/HubCanvasPage.tsx (hunk 1 above)
Comment (daystram) [reply fableous-fe/src/containers/HubCanvasPage.tsx]: fixed

**Comment by daystram** [comment]
Yes, that'd be the case.
> looks good, one qs, will we have popups on student side when they got an achivement, if so, will we do that by the hub sending a ws message?

**Review by nicolauscg — APPROVED** [review]

**Pull request merged**